### PR TITLE
docs(ecs/provisioners): update provisioners example

### PR DESCRIPTION
**Pull request by Lance52259**
**What this PR does / why we need it**:
Add a provisioners example for private key.

**Which issue this PR fixes**:
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
```
NONE
```

**Special notes for your reviewer**:

**Release note**:

```release-note

```

## PR Checklist

* [ ] Tests added/passed.
* [x] Documentation updated.
* [ ] Schema updated.

## Acceptance Steps Performed

```
NONE
```

### CODE REVIEW

File: examples/ecs/provisioners/README.md [hunk 1]
```diff
@@ -1,4 +1,75 @@
-# Using Provisioner over SSH to an ECS Instance
+# Using Provisioner over SSH to an ECS instance
 
-This example provisions an ECS instance with a Public IP Address
-and [runs a `remote-exec` provisioner](https://www.terraform.io/docs/provisioners/remote-exec.html) over SSH.
+This example provisions an ECS instance with a Public IP address and runs a `remote-exec` provisioner over SSH.
+
+## Using private key to remote ECS instance
+
+For `huaweicloud_compute_instance` resoruce, if If the user_data field is used, the `admin_pass` field will be invalid.
```
Comment (niuzhenguo) [inline examples/ecs/provisioners/README.md]: s/if If/if

Comment (niuzhenguo) [reply examples/ecs/provisioners/README.md]: if `user_data` specified, `admin_pass` setting will not take effect.

File: examples/ecs/provisioners/README.md [hunk 2]
```diff
@@ -1,4 +1,75 @@
...
+Therefore, we need to use a key-pair to log in to the ECS instance.
+For this example, we create a key-pair resource and specify the private key storage on local host.
```
Comment (niuzhenguo) [inline examples/ecs/provisioners/README.md]: s/storage/path

File: examples/ecs/provisioners/README.md (hunk 1 above)
Comment (Lance52259) [reply examples/ecs/provisioners/README.md]: The description has been updated.

File: examples/ecs/provisioners/README.md (hunk 2 above)
Comment (Lance52259) [reply examples/ecs/provisioners/README.md]: finished

**Pull request merged**